front-project/src/features-store/auth/auth.reducer.js

```diff
@@ -1,13 +1,33 @@
-import { TOGGLE_IS_LOGGED_IN } from "./auth.constants";
+import {
+  AUTH_ERR,
+  AUTH_FINISHED,
+  AUTH_SIGN_IN,
+  AUTH_SIGN_UP,
+  AUTH_STARTED,
+  TOGGLE_IS_LOGGED_IN,
+} from "./auth.constants";
 
 const initialState = {
```

Comment:
it's common pattern to store error and isLoading in store but in our app we need to store only "user", we don't need another fields